test/core/iomgr/fd_posix_test.cc

```diff
@@ -164,8 +164,7 @@ static void session_read_cb(void* arg,  // session
       // before notify_on_read is called.
       grpc_fd_notify_on_read(se->em_fd, &se->session_read_closure);
     } else {
-      grpc_core::Crash(absl::StrFormat("Unhandled read error %s",
-                                       grpc_core::StrError(errno).c_str()));
+      LOG(FATAL) << "Unhandled read error " << grpc_core::StrError(errno);
```

Comment:
For my learning, will LOG(FATAL) maintains locations information as grpc_core::Crash (http://shortn/_BojiaqEO6l)? Thanks